### PR TITLE
VueUI part 3 (1?) - The alias of sendToTopic

### PR DESCRIPTION
**Pull request by Karolis2011**
This is half of #10825

Aka ATCs - a short cut to calling sendToTopic without importing utils.

### CODE REVIEW

**Comment by Karolis2011** [comment]
!review

File: vueui/src/components/vui/item.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="item">
-    <!-- DEPRICATE PLEASE USE VUI-GROUP AND VUI-GROUP-ITEM -->
+    <!-- DEPRICATED PLEASE USE VUI-GROUP AND VUI-GROUP-ITEM -->
```
Comment (Arrow768) [inline vueui/src/components/vui/item.vue:3]:
```suggestion
    <!-- DEPRECATED PLEASE USE VUI-GROUP AND VUI-GROUP-ITEM -->
```

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by Geevies — DISMISSED** [review]
I don't know much about backend code, so I'm reviewing it on merit of having downloaded the branch and testing it on my private server.

- When interacting with the canister, I tried to increase the pressure output, the number jumped up before going back down. It only stayed when I maxed it out.
- I can't exit UIs with the big cross in the top right.
- Various buttons on modular computers at random just don't work.

I would suggest trying to fix these and then hosting your own private server to see if there's anything I missed.

**Review by JohnWildkins — APPROVED** [review]
I was only able to test it somewhat briefly, but it seems to work fine. Wasn't able to induce any out-of-the-ordinary behavior in VUIs.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**